configure.in

```diff
@@ -216,7 +216,7 @@ AC_OUTPUT(Makehelp)
 
 dnl AC_PTS_RUN_OK([perl -x -S ./ccdep.pl $CXX], [], [AC_MSG_ERROR(cannot compute depends)])
 rm -f Makedep
-AC_PTS_RUN_OK([make Makedep.force], [], [AC_MSG_ERROR(cannot compute depends)])
+AC_PTS_RUN_OK([${MAKE:-make} Makedep.force], [], [AC_MSG_ERROR(cannot compute depends)])
```

Comment:
Please also update the single corresponding line in `configure` accordingly. (Not rerunning `autoconf` to avoid the risk of breaking other things.)

Reply:
Done in 7d69627.